### PR TITLE
Jssolo

### PR DESCRIPTION
**Pull request by Ninjaphil24**
Dear Reviewer,
The following pull request represents my mobile menu with my first attempt at javascript.  
Please note:  Concerning the README file:  I have had 4 conflicting reviews (I think 4) so far concerning the matter of making the Readme file larger or smaller.  I request that you not spend any time with the readme file as I do not wish to change it back again.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,30 +7,44 @@
...
-    </nav>
-    <nav class="mobile">
-        <i class="fa fa-bars fa-2x" aria-hidden="true"></i>
-    </nav>    
+    <header class="toggle">
+        
+            <a href="#"><h1>Philip Modinos</h1></a>
+            <!-- <nav class="toggle-mobile">
+                <i class="fa fa-times fa-2x" aria-hidden="true"></i>
+            </nav> -->
```
Comment (andersonlebon) [inline index.html]: - [ ] To make this file clean I think it would be good to remove commented code.

File: index.html [hunk 2]
```diff
@@ -7,30 +7,44 @@
...
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                </div>
+                <div class="close">
+                    <div class="firstLine"></div>
+                    <div class="secondLine"></div>
+                </div>
+                    <!-- <i class="fa fa-bars fa-2x" aria-hidden="true"></i> -->
+            </nav>
```
Comment (andersonlebon) [inline index.html]: The same here 👇 
- [ ]  To make this file clean I think it would be good to remove commented code.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const navToggle = () => {
+  const burger = document.querySelector('.toggle');
+  const toggler = document.querySelector('.toggle');
```
Comment (andersonlebon) [inline main.js]: - [ ] Nice job 👏  on making the mobile menu functionality I can see everything is working 👏  however to apply javascript best practices I think it should be good to remove repeated code like here 👆  burger = toggle.

 Your code 
```js
  const navToggle = () => {
       const burger = document.querySelector('.toggle');
       const toggler = document.querySelector('.toggle');
       burger.addEventListener('click', () => {
          toggler.classList.toggle('active');
    });
};
```

 A better code 
```js
  const navToggle = () => {
       const toggler = document.querySelector('.toggle');
       toggler.addEventListener('click', () => {
          toggler.classList.toggle('active');
    });
};
```

File: index.html (hunk 2 above)
Comment (andersonlebon) [inline index.html]: ## [OPTIONAL]
- [ ] Kindly I would suggest you center items in your header. check here 👇 

Your design | the template 
-----|------
image | image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Ninjaphil24 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ninjaphil24** [comment]
1. Comments Removed
2. js fixed.
3. h1 centered.

**Review by t-yanick — APPROVED** [review]
Hi @Ninjaphil24, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**